mainnet-1/config/pre-genesis.json

```diff
@@ -3920,8 +3920,8 @@
         "community_pool_contract_config": {
           "voting_rules": {
             "voting_period": 21,
-            "quorum": "10.000000000000000000",
-            "threshold": "60.000000000000000000",
+            "quorum": "40.000000000000000000",
+            "threshold": "50.01000000000000000",
```

Comment:
Forgot to check these other contracts / values as well, good catch.

Will now do a full review of the contract parameters.